src/cmd/archiver.go

```diff
@@ -29,7 +23,8 @@ func newArchiverCommand() *cobra.Command {
 
 	cmd.AddCommand(newArchiverCompressCommand())
 	cmd.AddCommand(newArchiverDecompressCommand())
-	cmd.AddCommand(newToolsVersionCmd("mholt/archiver", archiverVersion))
+	// FIXME(mkcp): archivesVersion
+	cmd.AddCommand(newToolsVersionCmd("mholt/archives", archiverVersion))
```

Comment:
I believe you have the required `ldflags` to support making this change? 
```suggestion
	cmd.AddCommand(newToolsVersionCmd("mholt/archives", archivesVersion))
```

Reply:
Cleaning up some of the fixmes from the first pass that aren't actually in scope for this PR, standby.